qiskit_ibm_runtime/session.py

```diff
@@ -193,6 +193,38 @@ def backend(self) -> Optional[str]:
         """
         return self._backend
 
+    def status(self) -> Optional[str]:
+        """Return current session status."""
+        # TODO implement this when API changes are done
+        pass
+
+    def details(self) -> Optional[Dict[str, Any]]:
+        """Return session details."""
+        if self._session_id:
+            response = self._service._api_client.session_details(self._session_id)
+            if response:
+                return {
+                    "id": response.get("id"),
+                    "backend_name": response.get("backend_name") or response.get("backend"),
+                    "interactive_timeout": response.get("interactive_ttl")
+                    or response.get("interactiveSessionTTL"),
+                    "max_time": response.get("max_ttl") or response.get("maxSessionTTL"),
+                    "active_ttl": response.get("active_ttl") or response.get("activeTTL"),
+                    "state": response.get("state"),
+                    "accepting_jobs": response.get("accepting_jobs")
+                    or response.get("acceptingJobs"),
+                    "root_job": response.get("rootJob"),
+                    "root_job_started": response.get("rootJobStarted"),
+                    "last_job": response.get("lastJob"),
+                    "last_job_started": response.get("lastJobStarted")
+                    or response.get("last_job_started"),
+                    "last_job_completed": response.get("lastJobCompleted")
+                    or response.get("last_job_completed"),
+                    "activated_at": response.get("activatedAt") or response.get("started_at"),
+                    "closed_at": response.get("closedAt") or response.get("closed_at"),
+                }
```

Comment:
Need to work with the IQP API team on this because Jorge has a change up which is going to severely limit the number of fields they return from their API, like as written it doesn't include any of the ttl fields which I think is a mistake.

Reply:
At some point (hopefully in the near future) the cloud API should return an `activated_at` timestamp field and the quantum API should return an `activatedAt` field. Those will indicate when the session last went into the `active` state and can be used with the `active_ttl` to calculate how much time is remaining in the active window for the session (assuming the session has more jobs to run and doesn't go inactive after the interactive_ttl expires).

Reply:
The docstring should explain what these fields mean, or at least point to the docs that do.

Reply:
Updated docstrings in [5cbdd16](https://github.com/Qiskit/qiskit-ibm-runtime/pull/1119/commits/5cbdd16519cae64278adf5e5249f789891362495)